backend/pkg/server/user/user.go

```diff
@@ -84,3 +85,8 @@ func (u *usersImpl) Get(authHeader, secret string, tokenType TokenType) (*User,
 	}
 	return dbUser, nil
 }
+
+func (u *usersImpl) GetServiceAccount(tenantID uint64) (*User, error) {
+	fmt.Println("GetServiceAccount is not implemented in the OSS version")
```

Comment:
Just return an error:
`return nil, errors.New("not implemented")`
Returning `(nil, nil)` and only logging the error is an anti-pattern.